about.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./about.css">
+    <title>Capstone Project</title>
+</head>
+<body>
+    <header>
+      <nav>
+        <div class="bt">
+          <ul class="nav-menu1">
+            <li class="nav-item1">
+              <a class="nav-link1" href="#about"><img src="./Images/twitter (1).png" alt="twitter icon"></a>
+            </li>
+            <li class="nav-item1">
+              <a class="nav-link1" href="#contact"><img src="./Images/facebook.png" alt="facebook icon"></a>
+            </li>
+          </li>
+            <li class="nav-item1"                                                                         ">
+              <a class="nav-link1" href="#my-projects">Portfolio</a>
+            </li>
+            <li class="nav-item1">
+              <a class="nav-link1" href="#about">About</a>
+            </li>
+            <li class="nav-item1">
+              <a class="nav-link1" href="#contact">Contact</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
```

Comment:
[OPTIONAL]
- [ ] Kindly make sure you use the same design for the home Navbar and the about Navbar to follow the provided design
### Your design :point_down: 
![Screenshot from 2023-07-08 14-48-08](https://github.com/newtonah0/Microverse-Capstone-1/assets/91142432/4fae2a65-6d85-4dda-a057-c585db21b35a)
### What it is supposed to look like :point_down: 
![Screenshot from 2023-07-08 14-58-10](https://github.com/newtonah0/Microverse-Capstone-1/assets/91142432/e2540c2c-7c06-4011-b3d1-9ca58f3539be)

This is a requirement of the project but first reviewer didn't notice it. so I requested this as an optional but I highly recommend you to make the change